pubsub/test/iam.test.js

```diff
@@ -14,159 +14,278 @@
 'use strict';
 
 var proxyquire = require('proxyquire').noCallThru();
+var topicName = 'foo';
+var subscriptionName = 'bar';
+var policyMock = 'policy';
+var permissionsMock = 'permissions';
+
+function getSample () {
+  var subscriptionMock = {
```

Comment:
The two resources have the same methods. Using the same mock for two different resources would introduce additional cognitive overhead for any programmer who needs to read or update this test.